ui_framework/components/collapse_button/collapse_button.test.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { render,shallow } from 'enzyme';
+import { requiredProps } from '../../test/required_props';
+import sinon from 'sinon';
+
+import {
+  KuiCollapseButton,
+} from './collapse_button';
+
+describe('KuiCollapseButton', () => {
+  describe('Props', () => {
+    describe('direction', () => {
+      ['up','down','left','right'].forEach(direction => {
```

Comment:
And then we can import `DIRECTIONS` above, and loop through them here (note that I also changed the `test` name):

```jsx
      DIRECTIONS.forEach(direction => {
        describe(direction, () => {
          test(`renders the ${direction} class`, () => {
            const component = <KuiCollapseButton direction={direction} { ...requiredProps }/>;
            expect(render(component)).toMatchSnapshot();
          });
        });
      });
```